cfgov/v1/models/base.py

```diff
@@ -50,6 +50,14 @@ class Meta:
         verbose_name_plural = _("Tags")
 
 
+class CFGOVOwnedPages(TaggedItemBase):
```

Comment:
Inheriting from `TaggedItemBase` means that this field will reuse the same tag model as page categories and authors; this means that autocomplete will populate from those things, where ideally we'd want this to autocomplete only from other content owners. Could we consider creating a [custom tag model](https://docs.wagtail.org/en/stable/reference/pages/model_recipes.html#custom-tag-models) for this?

Reply:
Up for it but not certain whether it's something feasible to integrate easily into this PR vs. modifying in a later branch?

Reply:
It'd be better to do it here (if you're up for it) because changing the tag model means moving to a different database table, so it'll avoid having to do some kind of data migration later.